requirements.txt

```diff
@@ -8,4 +8,5 @@ requests-aws4auth==0.9
 django==1.8
 django-dotenv==1.3.0
 weighted-levenshtein==0.1
-ipython
\ No newline at end of file
+ipython
+googletrans==2.2.0
```

Comment:
We should not use googletrans and instead integrate official google translate api. Consider the ethical implications.

From the googletrans, readme
> DISCLAIMER: this is an unofficial library using the web API of translate.google.com and also is not associated with Google.

Reply:
Will change it before merging this in develop